test/spec/aic-shrinkwrap-spec.js

```diff
@@ -0,0 +1,65 @@
+let describeAicTest = describe;
```

Comment:
This file is a copy paste from above with only very little changes.

What does this test?